[CIAPP-3593] Explain tags query param in instructions for gitlab (#12994)
DataDog/documentation: content/en/continuous_integration/setup_pipelines/gitlab.md

: Specifies which environment (`env` tag) to attach to each span generated by the integration. Use this to differentiate between groups of GitLab instances (for example: staging or production).<br/>
**Default**: `none`

**Tags** (optional)

> : Specifies any custom tags to attach to each span generated by the integration. Provide one tag per line in the format: *key:value*.<br/>

Reviewer: ```suggestion
: Specifies any custom tags to attach to each span generated by the integration. Provide one tag per line in the format: `key:value`.<br/>
```

To set custom `env` or `service` parameters, add more query parameters in the webhooks URL: `&env=<YOUR_ENV>&service=<YOUR_SERVICE_NAME>`

> To set custom tags, provide the query parameter `tags`, with *key:value* pairs separated by commas. If the tag contains commas, quotes are needed. Example: `key1:value1,"key2: value with , comma",key3:value3`. (**This parameter needs URL encoding**, for the example: `&tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`)

Reviewer: ```suggestion
To set custom tags, add to the **Webhook URL** a URL-encoded query parameter `tags`, with `key:value` pairs separated by commas. If the tag pair contains any commas, quotes are needed. For example, to add `key1:value1,"key2: value with , comma",key3:value3`, the following string would need to be appended to the 
**Webhook URL**: `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
```

> To set custom tags, provide the query parameter `tags`, with *key:value* pairs separated by commas. If the tag  contains commas, quotes are needed. Example: `key1:value1,"key2: value with , comma",key3:value3`. (**This parameter needs URL encoding**, for the example: `&tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`)

Reviewer 2: ```suggestion
To set custom tags, provide the query parameter `tags`, with *key:value* pairs separated by commas. If the tag contains commas, quotes are needed. Example: `key1:value1,"key2: value with , comma",key3:value3`. (**This parameter needs URL encoding**, for the example: `&tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`)
```
